### PR TITLE
US217 Plot Styling

### PR DESCRIPTION
**Pull request by aneuhold**
### Overview
Implements User Story 217, which fixes the re-rendering on re-size and fixes the overlapping x-axis on a few plots.

Before:

image

After:

image

### Includes:
- client/package-lock.json 
- client/package.json 
- client/public/index.html 
- client/src/App.vue 
- client/src/components/Graph/Plot.vue 
- client/src/components/Graph/index.vue 
- client/src/utils/plot.js 

### Developer Checklist:
- [x] The code follows the Quality Policy file on Google Drive
- [x] My code has been developer-tested and includes unit tests. The unit tests are the ones that existed before for the graph.
- [x] I have considered proper use of exceptions
- [x] I have eliminated IDE warnings

### Notes:
- This fixes an issue where the home page would not scroll by removing an un-needed CSS rule.
- Another issue this fixes is where the number of models in the legend wouldn't go above 2. It works as expected now.

### Refs:
- [US 217](https://the-halo-mod.atlassian.net/jira/software/projects/HM/boards/1?selectedIssue=HM-217)

### CODE REVIEW

File: client/src/utils/plot.js [hunk 1]
```diff
@@ -152,12 +156,13 @@ export default (elementId, plotData) => {
   const datasets = Object.values(plotData.plot_data);
 
   // Build the color generator for the lines and legend
-  const colors = datasets.map((val, i, arr) => d3.rgb(
+  const colors = datasets.map((val, i) => d3.rgb(
     // The different color options are here: https://github.com/d3/d3-scale-chromatic
-    d3.interpolateCool(i / arr.length),
+    d3.interpolateCool(i / datasets.length),
   ).formatHex());
+  debug('colors are: ', colors);
```
Comment (aschnapp) [inline client/src/utils/plot.js:163]: Idk much about this debug function.. should we get rid of this before we merge?

**Review by aschnapp — APPROVED** [review]
Left a comment about debug. Approving this anyway.

File: client/src/utils/plot.js (hunk 1 above)
Comment (aneuhold) [reply client/src/utils/plot.js:163]: They don't take effect unless `debug.enabled = true` at the top of the file. [Link to the package](https://www.npmjs.com/package/debug).

**Review by vperuzzi — APPROVED** [review]
Looks great, Anton! 💯

**Pull request merged**